workflow/local-roles.rst

```diff
@@ -0,0 +1,12 @@
+Local Roles
```

Comment:
Some links to add: 
* http://docs.plone.org/working-with-content/collaboration-and-workflow/collaboration-through-sharing.html
* http://docs.plone.org/develop/plone/security/local_roles.html